featuretools/utils/pickle_utils.py

```diff
@@ -1,4 +1,4 @@
-import pickle
+import cloudpickle
```

Comment:
do you need to add cloudpickle to requirements?

Reply:
added